lib/Jsrt/JsrtDiag.cpp

```diff
@@ -159,15 +160,20 @@ CHAKRA_API JsDiagStopDebugging(
             jsrtDebugManager->ClearBreakpointDebugDocumentDictionary();
         }
 
-        *callbackState = jsrtDebugManager->GetAndClearCallbackState();
+        void* cbState = jsrtDebugManager->GetAndClearCallbackState();
+
+        if (callbackState != nullptr)
+        {
+            *callbackState = cbState;
+        }
 
         return JsNoError;
     });
 #endif
 }
 
 CHAKRA_API JsDiagGetScripts(
-    _Out_ JsValueRef *scriptsArray)
+    _Out_opt_ JsValueRef *scriptsArray)
```

Comment:
>_Out_opt_ [](start = 4, length = 9)

Why opt?

Reply:
You wanted it in JsDiagStopDebugging instead :-)

---
In reply to: [180273852](https://github.com/Microsoft/ChakraCore/pull/4958#discussion_r180273852) [](ancestors = 180273852)

Reply:
Sigh, sorry about that.  Fixed.